server/manager/src/main/java/org/apache/accumulo/manager/ManagerTime.java

```diff
@@ -74,8 +77,8 @@ public ManagerTime(Manager manager, AccumuloConfiguration conf) throws IOExcepti
    *
    * @return Approximate total duration this cluster has had a Manager, in milliseconds.
```

Comment:
This comment might need to be updated since it mentions the old return type

Reply:
Fixed in latest update